2022/39xxx/CVE-2022-39390.json

```diff
@@ -1,9 +1,8 @@
 {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng'}]}]}